common4j/src/main/com/microsoft/identity/common/internal/cache/IKeyBasedStorage.java

```diff
@@ -22,31 +22,33 @@
 // THE SOFTWARE.
 package com.microsoft.identity.common.internal.cache;
 
-import android.content.SharedPreferences;
+import com.microsoft.identity.common.java.util.ported.Predicate;
 
 import java.util.Iterator;
 import java.util.Map;
 
-
-public interface ISharedPreferencesFileManager {
+/**
+ * The interface for a key-value store.  This should
```

Comment:
This should...?

Reply:
yup.  Be roughly equivalent to android's Shared Preferences.  filled in.